esbuild/esbuild.js

```diff
@@ -1,18 +1,20 @@
 /* eslint-disable no-console */
-let path = require("path");
-let fs = require("fs");
-let glob = require("fast-glob");
-let esbuild = require("esbuild");
-let vue = require("esbuild-vue");
-let yargs = require("yargs");
-let cliui = require("cliui")();
-let chalk = require("chalk");
-let html_plugin = require("./frappe-html");
-let rtlcss = require('rtlcss');
-let postCssPlugin = require("esbuild-plugin-postcss2").default;
-let ignore_assets = require("./ignore-assets");
-let sass_options = require("./sass_options");
-let {
```

Comment:
I don't think `const` is better than `let` when dealing with objects. They are mutable anyway. This is just a matter of preference at this point.

I would avoid touching lines that don't "improve" anything unless it is a formatting improvement.

Reply:
Yes, objects are mutable. But at least `const` allows us to know that the object being referenced is still the same (contents may change).

Is it a matter of preference? Absolutely. A lot of people prefer `let` over `const` and vice-versa.
Is it _just_ a matter of preference? I don't think so.

Excerpt from StackOverflow:

> Using const when the variable's¹ value is not meant to change accomplishes a few things:
> 
> 1. It tells others reading your code that you don't intend the value to change.
> 
> 1. It gives you a nice proactive error if you change the code so it writes to that variable. (A decent IDE can flag this up proactively, but if not, you'll get the error when running the code.) You can then make an informed, intentional decision: Should you change it to let, or did you not mean to change that variable's value in the first place?
> 
> 1. It gives a hint to the JavaScript engine's optimizer that you won't be changing that variable's value. While the engine can frequently work that out through code analysis, using const saves it the trouble. (Caveat: I have no idea whether this is actually useful to the JavaScript engine. It seems like it would be, but runtime code optimization is a very complicated and sometimes non-intuitive process.)

@surajshetty3416 @gavindsouza thoughts?

Reply:
PS: I am only sharing my opinions here. It's perfectly okay to disagree. I'm sure we all want what we think is the best for the framework.

Reply:
Pointing out an object is a constant by using `const` makes sense to me. This seems more intuitive and an easy convention.

Apart from that, I think the refactor/style changes should be a separate commit. One commit to refactor the way assets are replaced and another for this change.